build/build.yml

```diff
@@ -101,6 +94,32 @@ steps:
     targetFolder: '$(Build.ArtifactStagingDirectory)\tarball\'
     flattenFolders: true
 
+- task: ArchiveFiles@2
+  displayName: Create netcore tarball
+  inputs:
+    rootFolderOrFile: '$(Build.ArtifactStagingDirectory)\tarball\'
```

Comment:
Not part of your change, but a bit confusing IMO that the 'tarball' folder is used to produce both a tarball/tgz, and a zip.

Reply:
yeah, I saw that too, forgot to make the change with this PR though. Will change in a separate PR.